### DISCUSSION TITLE
Magento-Sales Module Compatability

### DISCUSSION THREAD

**Discussion by wdevamar** [breezefront/community#151] [1. Help] [answered]
@vovayatsyuk ,
Please  Check sales/guest/form/ page that page is not working with Breeze Theme but working fine with Magento-Luma Theme.
Steps To Check Issue
1.Go to sales/guest/form/ page.
2.Change Find Order By to "Zip Code".
** There is no Change with dependent field from "Email" To "Billing ZIP Code".
URL :[https://www.awesomescreenshot.com/image/40608572?key=fbb501bc3e2a560ba54b812cfe47edcc](https://www.awesomescreenshot.com/image/40608572?key=fbb501bc3e2a560ba54b812cfe47edcc)

**Comment by vovayatsyuk** [comment, marked as answer]
Thank you. I've created an issue for this discussion: https://github.com/breezefront/module-breeze/issues/47

**Reply by vovayatsyuk** [reply]
Fixed in master brach. Will be published in the next release.
